packages/@okta/courage-for-signin-widget/src/CourageForSigninWidget.js

```diff
@@ -59,6 +59,16 @@ const events = {
 const TextBoxForSigninWidget = TextBox.extend({ events });
 const PasswordBoxForSigninWidget = PasswordBox.extend({ events });
 
+const Form = BaseForm.extend({
```

Comment:
why to make this change here at courage level ? Can we do this in `/v2/view-builder/internals/BaseForm` ?

Reply:
This option is not just for v2